packages/web/src/components/test-table/TestTable.tsx

```diff
@@ -391,7 +418,89 @@ export const TestTable = ({
     [rows]
   )
 
-  const renderContent = () => {
+  // Pagination Functions
+  const goToPage = useCallback(
+    (page: number) => {
+      if (page !== currentPage) setCurrentPage(page)
+    },
+    [currentPage]
+  )
+
+  const nextPage = useCallback(() => {
+    if (currentPage < maxPage) goToPage(currentPage + 1)
+  }, [currentPage, goToPage, maxPage])
+
+  const prevPage = useCallback(() => {
+    if (currentPage > 0) goToPage(currentPage - 1)
+  }, [currentPage, goToPage])
```

Comment:
If I quickly doubleclick the back button on page 2, can I go out of bounds?
I suspect we're okay because you won't regenerate the callback with `currentPage: 1` until the next render

Reply:
Oh, and it's guarded anyway. Sweet!